createRelease.sh

```diff
@@ -96,19 +96,35 @@ function zip_repo {
     echo "ZIPPING REPO: $REPO_NAME"
     echo $SEPARATOR
     
-    cd $COURSE_RELEASE_FOLDER
+    cd $RELEASE_DIR
```

Comment:
It's unclear why this was changed? Does it affect the resulting zip file in some way?